rdp-application/ssh_connection.go

```diff
@@ -22,9 +22,9 @@ func connect(sshServerHost string, sshServerPort int, user string, password stri
 	}
 	sshServerEndpoint := client.Endpoint{
 		Host: sshServerHost,
-		Port: sshServerPort,
+		Port: 2222,
```

Comment:
Just to confirm, this variable comes from accounts backend, it will be stay static?

Reply:
You are right... it should come from acocunts backend. Fixing it.